array-destructuring/exercise-2/exercise.js

```diff
@@ -70,3 +70,10 @@ let hogwarts = [
     occupation: "Teacher",
   },
 ];
+
+//let {firstName,lastName,house,pet,occupation} = hogwarts;
+hogwarts.filter(({firstName,lastName,house,pet,occupation }= hogwarts)=>{
```

Comment:
The `= hogwarts` part here doesn't do anything - the `filter()` callback with get the object representing each person in Hogwarts passed in in turn.